### PR TITLE
adding Content for README.md

### PR DESCRIPTION
**Pull request by ArthurFleischman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+## install python3
+
+## install pip3 (python package manager for python3)
+
+## recomended use venv (python virtual environment something like docker container for python project) doc bellow:
+> ## https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
+## to activate venv use: 
+> ## source venv/bin/activate (unix base)
+> ## .\env\Scripts\activate (windows)
```
Comment (HenriqueDerosa) [inline README.md]: This should be bash code, right?
Using `>` means it's a quote, not a code. Did you try it with any markdown preview?
you should be using tripple `

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/arthur/VisualStudioProjects/Python/ranked-voting/venv/bin/python3.8"
```
Comment (HenriqueDerosa) [inline .vscode/settings.json]: This is your local path, it depends on the user's operation system as well.
If we must keep this file we could use an environment variable? 🤔

File: README.md (hunk 1 above)
Comment (ArthurFleischman) [reply README.md]: Oh... i forgot this command, ill change

File: .vscode/settings.json (hunk 2 above)
Comment (ArthurFleischman) [reply .vscode/settings.json:2]: Ill change the interpreter path... sorry haha

Comment (ArthurFleischman) [reply .vscode/settings.json:2]: Ill try to use the relative path

Comment (ArthurFleischman) [reply .vscode/settings.json]: now i took deeper observation of it, and i forgot to add .vscode stuff in the .gitignore, my bad

**Comment by ArthurFleischman** [comment]
now i think everything will be ok

**Comment by lucasmontano** [comment]
Thanks for your PR! But I’m not sure why you’re addressing framework implementation here, as we still not over with the issue #2 

IMO: It seems like to much for one PR that was supposed to add the initial Readme

**Comment by ArthurFleischman** [comment]
> Thanks for your PR! But I’m not sure why you’re addressing framework implementation here, as we still not over with the issue #2
> 
> IMO: It seems like to much for one PR that was supposed to add the initial Readme

Oh, ok. Ill slow down...

**Comment by ArthurFleischman** [comment]
now its on our parameters

**Comment by fernandovmp** [comment]
I think open a new PR will be better, since merge this will pollute the commit history with an unused flask implementation and with the venv directory (which should be added to gitignore)

**Comment by ArthurFleischman** [comment]
now its just README.md for my pull request, @HenriqueDerosa @lucasmontano.

**Comment by ArthurFleischman** [comment]
hope its what you want...

**Comment by fernandovmp** [comment]
When i say to open a new PR, i mean create a new branch for it. If you look at commits section you'll see the entire history where only the lasts commits are relevance, and merge all of then isn't the best option. 

Other option is rebase all the branch, but this will be harder than create a new PR.

**Comment by ArthurFleischman** [comment]
@fernandovmp ill do it

**Comment by ArthurFleischman** [comment]
but, is the content ok?
- venv 
- setup
- run

**Comment by fernandovmp** [comment]
@ArthurFleischman to the readme? I belive yes, but i did't have expirience with python projects yet, so i don't know how is the ideal setup guide.

**Comment by ArthurFleischman** [comment]
@fernandovmp i just putted an extra layer, thats VENV looks like a dcoker container. so nobody needs to install dependencies... just the first person

**Comment by fernandovmp** [comment]
Looks good, however I found [here](https://medium.com/@jtpaasch/the-right-way-to-use-virtual-environments-1bc255a0cba7) that the venv directory should be ignored and a requiriments.txt created.

**Comment by ArthurFleischman** [comment]
@fernandovmp i saw it too, but is the default .gitignore, we have no reason to dont use it,
with i command we start using  the python interpreter inside the VENV and also all the libs, and with other, we return to normal. we also have the switch button in vscode to select venv.
Screenshot from 2020-05-11 21-01-17

**Comment by fernandovmp** [comment]
Huumm... I don't know exactly about it, the venv/bin/activate have a path of your local machine, but if we switch this to an relative path this should not have to create any conflict, also I notice that the previous branch have a 2mb size and I think that the venv directory is the problem. However I haven't enough knowledge about the venv so I will leave the decision of use it in setup (without the gitignore) for you or any other with experience.

**Comment by ArthurFleischman** [comment]
@fernandovmp this 2mb of filles will need to be created in everybody setup, we are just providing a default one with everything on, so less setup + code... its my personal opinion.

**Comment by ArthurFleischman** [comment]
@fernandovmp move this conversation to the new pull request

**Pull request closed without merging**